modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -21,7 +21,7 @@ In order to collect traffic from a cluster with a Single Root I/O Virtualization
 .Configure `FlowCollector` for SR-IOV monitoring
```

Comment:
To be honest, I do not know 🤔 I guess I thought I needed them because of the `.Configure `FlowCollector` for SR-IOV monitoring` YAML heading. I don't think its hurting anything per say, but it may also be pointless.